internal/cluster/cluster.go

```diff
@@ -22,6 +23,7 @@ func (c Cluster) String() string {
 	tmp = append(tmp, fmt.Sprintf("MetricsURL: %s", c.MetricsURL))
 	tmp = append(tmp, fmt.Sprintf("LoggingURL: %s", c.LoggingURL))
 	tmp = append(tmp, fmt.Sprintf("AppDNS: %s", c.AppDNS))
+	tmp = append(tmp, fmt.Sprintf("Type: %s", c.Type))
```

Comment:
I think it is better to use `bytes.Buffer` and `WriteString` to create the string.

Reply:
I feel that's over optimization. String() isn't called that much and fmt.Sprintf covers the usecase for use.